classes/01class/exercises/c01-e04/starkmatt/c01-e04.txt

```diff
@@ -0,0 +1,13 @@
+#Created IAM Role for S3FullAccess, went back into my Inital Public instance and changed IAM Rule to include S3FullAccess
```

Comment:
as a challenge, can you create a custom policy and add just the minimal permission needed instead of using S3FullAccess? I will mark this pull request as approved, but feel free to submit more work into that

Reply:
Roger that!

Reply:
Done!
Changed the IAM Role from **S3FullAccess** to **AmazonS3ReadOnlyAccess**